src/braze/tags/connectedContent.ts

```diff
@@ -49,15 +59,36 @@ export default <ITagImplOptions>{
       contentType = this.options.content_type || 'application/x-www-form-urlencoded'
     }
 
+    const headers = {
+      'User-Agent': 'brazejs-client',
+      'Content-Type': contentType,
+      'Accept': this.options.content_type
+    }
+    if( this.options.headers ) {
+      // Add specified headers to the headers
+      const keys = Object.keys(this.options.headers);
+      for(let i = 0; i < keys.length; i++ ) {
+        let key = keys[i];
+        if( this.options.headers[key] ) {
+          headers[key] = await this.liquid.parseAndRender(this.options.headers[key], ctx.getAll())
+        } 
+      }
```

Comment:
```suggestion
      Object.keys(this.options.headers).forEach((key) => {
        headers[key] = await this.liquid.parseAndRender(this.options.headers[key], ctx.getAll())
      })
```

Reply:
this could address the test coverage dropping error too

Reply:
This loop runs into an issue where the `forEach` block is not async, and so the `await` in the foreach isn't valid. Making the foreach async then requires it to be awaited on, and I'm not sure how to format that with promises and what not. I'm not well versed with JS, so maybe you could make a recommendation?

Reply:
ah sorry I overlooked the `await`, yes this fine. but can we remove the `if( this.options.headers[key] )`? or provide a test for it as it's not covered as per https://coveralls.io/builds/34508425/source?filename=src%2Fbraze%2Ftags%2FconnectedContent.ts#L72

Reply:
for async for loop, we have 2 ways:

```
for (const key of Object.keys(this.options.headers)) {
    headers[key] = await this.liquid.parseAndRender(this.options.headers[key], ctx.getAll())
  }
```

or
 
```
await Promise.all(Object.keys(this.options.headers).map(async (key) => {
    headers[key] = await this.liquid.parseAndRender(this.options.headers[key], ctx.getAll())
  }));
```